mapper.go

```diff
@@ -0,0 +1,265 @@
+package tupleconv
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/tarantool/go-tarantool/datetime"
+	"github.com/tarantool/go-tarantool/decimal"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// Mapper is a mapper from a certain type to any.
+type Mapper[T any] interface {
+	Map(src T) (any, error)
+}
+
+// Interface validations.
+var (
+	_ Mapper[string] = (*BoolParser)(nil)
+	_ Mapper[string] = (*UIntParser)(nil)
+	_ Mapper[string] = (*IntParser)(nil)
+	_ Mapper[string] = (*FloatParser)(nil)
+	_ Mapper[string] = (*DecimalParser)(nil)
+	_ Mapper[string] = (*UUIDParser)(nil)
+	_ Mapper[string] = (*StringParser)(nil)
+	_ Mapper[string] = (*DateTimeParser)(nil)
```

Comment:
What's about interval? https://github.com/tarantool/go-tarantool/blob/master/datetime/interval.go

Reply:
The commit with interval support added.